docs/BreakingChanges/029 The Replace method in OData URLs is disabled by default.md

```diff
@@ -0,0 +1,22 @@
+## 29: The Replace method in OData URLs is disabled by default.
```

Comment:
{
  "Id": "29",
  "Title": "The Replace method in OData URLs is disabled by default.",
  "Details": "Beginning in the .NET Framework 4.5, the Replace method in OData URLs is disabled by default. When OData Replace is disabled (now by default), any user requests including replace functions (which are uncommon) will fail.",
  "Suggestion": "If the replace method is required (which is uncommon), it can be re-enabled through a [config settings](https://msdn.microsoft.com/en-us/library/system.data.services.configuration.dataservicesfeaturessection.replacefunction.aspx). However, an enabled replace method can open security vulnerabilities and should only be used after careful review.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#wcf",
  "Markdown": null,
  "ApplicableApis": [
    "T:System.Data.Services.DataService`1"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
[x] T:System.Data.Services.DataService`1